docs/user-guide/share.md

```diff
@@ -33,12 +33,19 @@ The **Social** section allows the user to share the resource on the most common
 
 ## Embed
 
-The **Embed** section provides to the user the needed snippets, **embedded code** or the **MS APIs** to embed MapStore in a third party web page.
+The **Embed** section provides to the user the needed snippets, **embedded code** or the **MS APIs** (only available for *maps*) to embed MapStore in a third party web page.
 
 <img src="../img/share/embed.jpg" class="ms-docimage"  style="max-width:400px;"/>
 
-!!!note
-    At the moment the embed section is available only for *maps*.
+In addition, [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) provides two options for customized the *embedded code*:
+
+* For maps, the user can add [TOC](toc.md) to the embedded map by enabling the **Shown TOC**
+
+<img src="../img/share/embed_maps.jpg" class="ms-docimage"  style="max-width:400px;"/>
+
+* For dashboards, the user can view the [connections](connecting-widgets.md) between the widgets on the embedded dashboard by enabling the **Show connections**
```

Comment:
* For dashboards, the user can show the [connections](connecting-widgets.md) between widgets on the embedded dashboard by enabling the **Show connections**